POSTFIX_CERTS_GENERATION.md

```diff
@@ -0,0 +1,102 @@
+# Steps for generation self signed SSL key for postfix
+Let's create a self signed SSL key in cert directory.
```

Comment:
You mean 'self-signed certificate', not key. Also, explain where the 'certs' directory is supposed to be.